### PR TITLE
Alerting: Display last & next rule eval date plus eval duration

### PR DESCRIPTION
**Pull request by VikaCep**
**What is this feature?**

Displays the last and next evaluation date plus the last evaluation duration for an alert rule.

**Next evaluation date** is shown in a new column for each rule in the list. The table shows the humanized version of the date, and on hover the full date is displayed. 

**Last evaluation date** and **last evaluation duration** are shown once the details for the selected rule are expanded. Again, full date formats are shown on hover.

**Why do we need this feature?**

This will provide more clarity for users to understand evaluation times.

**Who is this feature for?**

All users.

**Which issue(s) does this PR fix?**:

Fixes https://github.com/grafana/alerting-squad/issues/434

**Special notes for your reviewer**:

image
image

### CODE REVIEW

File: public/app/features/alerting/unified/utils/time.ts [hunk 1]
```diff
@@ -99,3 +99,7 @@ export function parsePrometheusDuration(duration: string): number {
 
   return totalDuration;
 }
+
+export const isNullDate = (date: string) => {
```
Comment (soniaAguilarPeiron) [inline public/app/features/alerting/unified/utils/time.ts:103]: I guess we know that null dates will come with this value?

**Review by soniaAguilarPeiron — APPROVED** [review]
LGTM! 
Great job!!! 🎉

File: public/app/features/alerting/unified/utils/time.ts (hunk 1 above)
Comment (VikaCep) [reply public/app/features/alerting/unified/utils/time.ts:103]: Yes, I verified that in some cases we get those kind of dates (like for paused alert rules) and trying to format them would display something completely wrong

**Comment by VikaCep** [comment]
Thanks for reviewing @soniaAguilarPeiron! <3

**Pull request merged**